src/Pages/CharacterDetail/CharacterDetail.js

```diff
@@ -0,0 +1,73 @@
+import {useRoute} from '@react-navigation/native';
+import React from 'react';
+import {Text, View, ScrollView, Image, Linking} from 'react-native';
+import {useTranslation} from 'react-i18next';
+
+import styles from './CharacterDetail.styles';
+import useFetch from '../../hooks/useFetch';
+import Button from '../../Components/Button/Button';
+import DetailCard from '../../Components/DetailCard';
+
+const CharacterDetail = () => {
+  const route = useRoute();
+  const {id} = route.params;
+  const {data} = useFetch(`characters/${id}`);
+  const {t, i18n} = useTranslation();
+
+  async function openLink(url) {
+    await Linking.openURL(url);
+  }
+
+  return (
+    <ScrollView>
+      <View style={styles.animated_header_component}>
+        <Image
+          style={styles.image}
+          source={{
+            uri: 'https://static.wikia.nocookie.net/marveldatabase/images/e/e1/The_Marvel_Universe.png/revision/latest/scale-to-width-down/630?cb=20110513164401',
+          }}
+        />
+      </View>
+      {data &&
```

Comment:
Daha önce de söylemiştim, bu şekilde kontrol yapmayın. Boolean yapısında çevirin.
```suggestion
      {!!data &&
```